src/table/settings/economy_settings.ini

```diff
@@ -300,3 +300,16 @@ str      = STR_CONFIG_SETTING_INFRASTRUCTURE_MAINTENANCE
 strhelp  = STR_CONFIG_SETTING_INFRASTRUCTURE_MAINTENANCE_HELPTEXT
 post_cb  = [](auto) { InvalidateWindowClassesData(WC_COMPANY_INFRASTRUCTURE); }
 cat      = SC_BASIC
+
+[SDT_VAR]
+var      = economy.cargo_scale
+type     = SLE_UINT16
+flags    = SF_NO_NETWORK
+def      = 100
+min      = 15
+max      = 400
+interval = 25
```

Comment:
I always have a bit of issue with an interval that is contradicting the min/def. What I mean: def - n * interval != min. It is not wrong, and the system can handle it fine, but it just feels .. meh :P Do with this what-ever you want :)

Reply:
I am mostly checking if you did this deliberately or not :) Pick one, I am fine with what-ever. As long as it is a conscious decision, I am fine with it :)